frontend/src/components/Blog/Story.tsx

```diff
@@ -127,10 +130,11 @@ const ActionBox = () => {
   const beginEditStory = () => {
     navigate(`/edit/${blog.id}`);
   };
+
   return (
     <div className="text-slate-500 py-2 items-center justify-between flex border-y border-main">
       <div className="text-sm">
-        <ClapButton clapCount={blog?.claps?.length || 0} handleClap={likeBlog} />
+        <ClapButton clapCount={blog?.claps?.length || 0} handleClap={likeBlog} darkMode={theme === 'dark'} />
```

Comment:
Instead of passing darkMode as a prop, you can call useTheme inside the <ClapButton /> component.